dist/obs_worker_manager

```diff
@@ -0,0 +1,491 @@
+#!/bin/sh
+
+# Is the worker running on Fedora >= 17 ?
+# It lacks the complete path to build other distros then
+bash_bin=`type -p bash`
+if [ "$bash_bin" != "${bash_bin#/usr}" ]; then
+    export PATH="/bin:$PATH"
+fi
+
+if test -e /etc/sysconfig/proxy; then
+  . /etc/sysconfig/proxy
+  if [ "$PROXY_ENABLED" != "no" ]; then
+    export http_proxy="$HTTP_PROXY"
+    export HTTPS_PROXY
+    export NO_PROXY
+  fi
+fi
+if test -e /etc/sysconfig/obs-server; then
+    # optional on workers
+    . /etc/sysconfig/obs-server
+fi
+# This file may still exist from OBS 2.1 and before.
+if test -e /etc/sysconfig/obs-worker; then
+    . /etc/sysconfig/obs-worker
+fi
+
+# Determine the base and follow a runlevel link name.
+base=${0##*/}
+link=${base#*[SK][0-9][0-9]}
+
+# Preconfigured by obsstoragesetup runlevel setup script
+if [ -f /etc/buildhost.config ];then
+    . /etc/buildhost.config
+fi
+
+if [ -z "$OBS_WORKER_DIRECTORY" ]; then
+    OBS_WORKER_DIRECTORY="/var/cache/obs/worker"
+fi
```

Comment:
I believe you can use parameter expansion to provide defaults. I think something like this should work
```suggestion
OBS_WORKER_DIRECTORY="${OBS_WORKER_DIRECTORY:=/var/cache/obs/worker}"
```

Reply:
The same can be done for the variables below

Reply:
dist/obs_worker_manager is just a copy of dist/obsworker, modified to use systemd instead of screen. It is not refactored on purpose, to make it clearer what the needed changes are.

Further refactorings of dist/obs_worker_manager could be left for following pull requests. Or also, refactorings of the current code of dist/obsworker can be created in other pull requests and after being merged in master, be inherited in this pull request.

Reply:
A follow up would be cool. Allright then!